.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: python
 python: 3.7
 cache: pip
-before_install: pip install --upgrade pip setuptools
+before_install: 
+  - pip install --upgrade pip setuptools
```

Comment:
Why this change?  How is it different than the current code?

Reply:
Formatting leftovers during previous attempts of testing builds. I'll revert this.